doc/release-process.md

```diff
@@ -8,6 +8,7 @@ Release Process
 * Update translations see [translation_process.md](https://github.com/bitcoin/bitcoin/blob/master/doc/translation_process.md#synchronising-translations).
 * Update release candidate version in `configure.ac` (`CLIENT_VERSION_RC`).
 * Update manpages (after rebuilding the binaries), see [gen-manpages.py](https://github.com/bitcoin/bitcoin/blob/master/contrib/devtools/README.md#gen-manpagespy).
+* Generate a bitcoin.conf file, see [gen-bitcoin-conf.sh](https://github.com/bitcoin/bitcoin/blob/master/contrib/devtools/README.md#gen-bitcoin-confsh).
```

Comment:
I would guess it also needs to be added to the repository, and committed?

Reply:
it does need to be committed for the build script to be aware of it. I went ahead and added a placeholder file (same as `doc/man/` and added a note in the docs to make it clear that the file does need to be committed. as a bonus, having a placeholder file should make it easier for users to discover the script

Reply:
Sounds like a good solution to me, thanks!